src/services/chris_integration.tsx

```diff
@@ -46,6 +46,12 @@ export interface BackendPollResult {
   error?: Error;
 }
 
+export interface pluginData {
+  title: string,
+  status: string,
+  plugin_name: string
```

Comment:
Stick to camelCase for attribute names

```suggestion
  pluginName: string
```